### PR TITLE
ft: ZENKO-1282 add ingestion schedule resume routes

### PR DESCRIPTION
**Pull request by philipyoo**
goes with https://github.com/scality/Arsenal/pull/602

### CODE REVIEW

File: lib/api/BackbeatAPI.js [hunk 1]
```diff
@@ -1118,25 +1110,35 @@ class BackbeatAPI {
             });
             this._redisPublisher.publish(channel, message);
         });
-        this._logger.info(`${service} service resumed for locations ${sites}`);
+        let message;
```
Comment (bennettbuchanan) [inline lib/api/BackbeatAPI.js]: Minor: Can save some lines and this var by logging in the if...else blocks.

File: lib/api/BackbeatAPI.js [hunk 2]
```diff
@@ -132,12 +132,9 @@ class BackbeatAPI {
     _updateConfigSites() {
         const bootstrapList = this._repConfig.destination.bootstrapList;
         const sites = bootstrapList.map(item => item.site);
-        // TODO-FIX: will change based on ingestion configs
-        const ingestionSites = this._ingestionConfig.sources.map(s => s.name);
-
-        // all sites except ingestion sites
-        this._crrSites = sites.filter(s => !ingestionSites.includes(s));
-        this._ingestionSites = ingestionSites;
+        // TODO-FIX: Separate ingestion and replication sites;
```
Comment (bennettbuchanan) [inline lib/api/BackbeatAPI.js:135]: Are we just waiting on a config update?

Comment (philipyoo) [reply lib/api/BackbeatAPI.js:135]: yup, currently the means of getting the config update is unclear, but the filter by service should be applied eventually, so leaving these comments

**Comment by philipyoo** [comment]
@bert-e wait

**Review by jonathan-gramain — APPROVED** [review]

**Review by anurag4DSB — APPROVED** [review]

**Review by rahulreddy — APPROVED** [review]

**Comment by philipyoo** [comment]
@bert-e approve

**Pull request merged**